### ISSUE TITLE
Code Suggestion 

### ISSUE THREAD

**Issue by JeffersonGibin** [serverless-dna/effe#17]
@walmsles 

I would like to leave 2 suggestions, please analyze it.

### Suggestion 1

What do you think about moving or adding comments to the code of each interface field specifically to the interface Adapter.?

### E.g:

```typescript
export interface IConnector {
    /** 
       What is it ?
    **/
    name: string;
}
```
### IDE
![image](https://github.com/user-attachments/assets/d9fc37e6-9b59-4864-b231-a2a929abe455)

This way, the developers can receive suggestions in your IDE when over mouse about the property.

### Suggestion 2

If you are not going to generate automatic documentation based on the code, it may not be necessary to have so many comments in the implementations because after that you will have documentation for the developers.

### My Opinion

I think comments on the interface are important to guide developers during implementation but it is questionable.

**Comment by walmsles** [comment]
I agree - both are important.  I have used TypeDoc commenting in the EventHub class file already for this reason.  Feel free to submit a PR

**Comment by JeffersonGibin** [comment]
@walmsles 

Close the issue pleas. It is solved in the PR #20
